presto-hive/src/main/java/com/facebook/presto/hive/OrcFileWriter.java

```diff
@@ -28,6 +28,7 @@
 import com.facebook.presto.orc.OrcWriter;
 import com.facebook.presto.orc.OrcWriterOptions;
 import com.facebook.presto.orc.OrcWriterStats;
+import com.facebook.presto.orc.WriterStats;
```

Comment:
This will fail the check style by having redundant imports

Reply:
Fixed. I compiled specific modules and did not compile the presto-hive. Will compile more modules and will watch out for the next commits.